inst/schema/ValidateInputResponse.schema.json

```diff
@@ -1,3 +1,14 @@
 {
-	"type": "string"
+  "type": "object",
```

Comment:
I don't love this - can we include a "role" field or similar in here too perhaps, being the "type" of the file (i.e., an enum of `pjnz`, `shapefile`, etc) and then a separate subschema for each?  You could enforce shared schema with an `allOf` and adding in

```
{
  "type": "object",
  "properties": {
    "filename": { "$ref": "FileName.schema.json" },
    "role": {
      "type": "string",
    }
    "data": {}
  },
  "additionalProperties": false,
  "required": [ "filename", "role", "data" ]
}
```

Then a schema for each type

```
{
  "type": "object",
  "properties": {
    "filename": {},
    "role": {
      "const": "pjnz"
    }
    "data": {
      "$ref": "PjnzResponseData.schema.json"
    }
  },
  "additionalProperties": false,
  "required": [ "filename", "role", "data" ]
}
```

The problem with the current approach is that we'd never want to accidentally get a pjnz response for a shapefile validation for example (not that that would happen)